### PR TITLE
MEN-6003 Protect user updates from concurrent updates 

### PR DESCRIPTION
**Pull request by alfrunes**

### CODE REVIEW

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline

**Review by kjaskiewiczz — APPROVED** [review]
just a comment - ETag support looks correct, but at the same time feels a bit complex;
in other places we're using random strings; should we change that too?
maybe it's a good idea to move part of this to go-lib-micro and unify implementation (not as part of this task, but some follow-up)?

File: api/http/api_useradm.go [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-//    Unless required by applicable law or agreed to in writing, software
-//    distributed under the License is distributed on an "AS IS" BASIS,
-//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-//    See the License for the specific language governing permissions and
-//    limitations under the License.
+//	Unless required by applicable law or agreed to in writing, software
+//	distributed under the License is distributed on an "AS IS" BASIS,
+//	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//	See the License for the specific language governing permissions and
+//	limitations under the License.
```
Comment (tranchitella) [inline api/http/api_useradm.go:13]: What about a separate commit for this?

**Review by tranchitella — APPROVED** [review]

File: api/http/api_useradm.go (hunk 1 above)
Comment (alfrunes) [reply api/http/api_useradm.go:13]: Sorry, this was an automatic format on save. Can I please keep it as is to not redo the merge in enterprise?

**Pull request merged**